packages/telltime/telltime.0.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Cli tool for interacting with Daypack-lib components"
+description: """
+Telltime relies on Daypack-lib, which is WIP. Do not use either for anything serious.
+"""
+maintainer: "Darren Ldl <darrenldldev@gmail.com>"
+authors: "Daypack developers"
+license: "MIT"
+homepage: "https://github.com/daypack-dev/telltime"
+bug-reports: "https://github.com/daypack-dev/telltime/issues"
+dev-repo: "git+https://github.com/daypack-dev/telltime"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "1.11.0"}
```

Comment:
```suggestion
  "dune" {>= "2.6"}
```

Your dune-project file requires dune 2.6

Reply:
Can you upstream the change?

Reply:
Aha, didn't realise this project used `lang dune 2.6`. Thanks and done.